src/gs-overview-page.c

```diff
@@ -1068,10 +1068,9 @@ gs_overview_page_setup (GsPage *page,
 		gtk_container_add (GTK_CONTAINER (priv->box_popular), tile);
 	}
 
```

Comment:
I agree. I left it there because I found it was not working during the rebase and intended to discuss it upstream, but I think it's fine either way to keep it at the beginning.